cigaradvisor/styles/styles.css

```diff
@@ -131,4 +131,9 @@ main .section[data-layout="50/50"]  .imagecta.block > a{
   main .section[data-layout="50/50"] .imagecta.block>a img {
     border: 1px solid #000;
   }
-}
\ No newline at end of file
+
+  /* removing border for 2 column videos section */
+  main .two-column-section[data-layout="50/50"] .imagecta.block>a img {
```

Comment:
Global styling shouldn't have block-level declarations.

if the block needs to change based on a wrapping context, move this to that CSS file.

i.e:  It's ok for a block to put global qualifiers in its CSS, it's not ok for a Global CSS to use any `.block` qualifiers